src/engraving/dom/segment.cpp

```diff
@@ -1086,6 +1086,16 @@ void Segment::checkEmpty() const
     }
 }
 
+double Segment::xPosInSystemCoords() const
+{
+    return x() + measure()->x();
```

Comment:
Shouldn't this return mutldata()->posX()? It's strange that the set and get functions work differently